config/dev.exs

```diff
@@ -115,6 +113,8 @@ config :archethic, ArchEthic.SharedSecrets.NodeRenewalScheduler,
 config :archethic, ArchEthic.P2P.Endpoint,
   port: System.get_env("ARCHETHIC_P2P_PORT", "3002") |> String.to_integer()
 
+config :archethic, faucet: true
```

Comment:
I think you can use 

```elixir
config :archethic, ArchEthicWeb.FaucetController, 
  enabled: true
``` 
instead of `config :archethic, faucet: true`

to more consistent with the rest and also it will only be related to the controller

Reply:
I will be using faucet in both router and LayoutView. Can I mention multiple modules one config call?

Reply:
For the LayoutView, because your route is related to the FaucetController, I think you can keep it inside the FaucetController configuration